rocky/assets/js/boefjeSetupRunOn.js

```diff
@@ -0,0 +1,36 @@
+function toggleElements(value) {
```

Comment:
If feel this name is a bit too general. Lets make it more specific to avoid collisions in the future.